qiskit/algorithms/evolvers/real/trotterization/trotter_qrte.py

```diff
@@ -0,0 +1,225 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""An algorithm to implement a Trotterization real time-evolution."""
+
+from typing import Union, Optional
+
+from qiskit import QuantumCircuit
+from qiskit.algorithms.aux_ops_evaluator import eval_observables
+from qiskit.algorithms.evolvers import EvolutionProblem, EvolutionResult
+from qiskit.algorithms.evolvers.real.real_evolver import RealEvolver
+from qiskit.opflow import (
+    SummedOp,
+    PauliOp,
+    CircuitOp,
+    ExpectationBase,
+    CircuitSampler,
+    PauliSumOp,
+    StateFn,
+)
+from qiskit.circuit.library import PauliEvolutionGate
+from qiskit.providers import Backend, BaseBackend
+from qiskit.quantum_info import SparsePauliOp, Pauli
+from qiskit.synthesis import ProductFormula, LieTrotter
+from qiskit.utils import QuantumInstance
+
+
+class TrotterQRTE(RealEvolver):
+    """Class for performing Quantum Real Time Evolution using Trotterization.
```

Comment:
This could just be `Quantum Real Time Evolution using Trotterization.` some things should be self-evident via the language like saying its a Class.